src/bric.c

```diff
@@ -46,63 +46,6 @@ void sigwinch_handler()
 	editor_refresh_screen();
 }
 
-// Low level terminal handling
-/*
-void disable_raw_mode(int fd)
-{
-        // dont bother checking the return value as its too late
-        if (Editor.rawmode) {
-                tcsetattr(fd, TCSAFLUSH, &orig_termios);
-                write(fd, "\033[2J\033[H\033[?1049l", 15);
-                Editor.rawmode = 0;
-        }
-}
-*/
-
-void editor_at_exit(void)
-{
-        disable_raw_mode(STDIN_FILENO, &orig_termios, &Editor);
-}
-
-int enable_raw_mode(int fd)
-{
-        struct termios raw;
-
-        if(Editor.rawmode) return 0; //already enabled
-        if(!isatty(fd)) goto fatal;
-        atexit(editor_at_exit);
-        if(tcgetattr(fd, &orig_termios) == -1) goto fatal;
-
-        raw = orig_termios; // modify the original mode
-        /* input modes: no break, no CR to NL, no parity check, no strip char,
-         *      * no start/stop output control. */
-        raw.c_iflag &= ~(BRKINT | ICRNL | INPCK | ISTRIP | IXON);
-
-        // output modes - disable post processing
-        raw.c_oflag &= ~(OPOST);
-
-        //control modes - set 8 bit chars
-        raw.c_cflag |= (CS8);
-
-        //local modes, choing off, canonical off, no extended functions, no signal chars (, etc)
-        raw.c_lflag &= ~(ECHO | ICANON | IEXTEN | ISIG);
-
-        //control chars - set return condition: min number of bytes and a timer
-        raw.c_cc[VMIN] = 0; // return each byte, or zero for a timeout
-        raw.c_cc[VTIME] = 1; //100ms timeout
-
-        //put terminal in raw mode after flushing
-        if(tcsetattr(fd, TCSAFLUSH, &raw) < 0) goto fatal;
-        //if(write(fd, "\033[?1049h\033[2J\033[H", 15) != 15) goto fatal;
-        Editor.rawmode = 1;
-        return 0;
-
-fatal:
-        errno = ENOTTY;
-        return -1;
-}
-
-
```

Comment:
I've delete this unused code, 'cause its content by `handling.c` and `handling.h`